src/styles/index.css

```diff
@@ -0,0 +1,38 @@
+body {
+  font-family: Arial, sans-serif;
+  background-color: #f4f4f4;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+}
+
+.calendar {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+  gap: 10px;
+  width: 80%;
+  max-width: 600px;
+}
+
+.day {
+  background-color: #ffffff;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  height: 100px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 24px;
+  position: relative;
+}
+
+.day::before {
+  content: attr(data-day);
+  position: absolute;
+  top: 10px;
+  left: 10px;
+  font-size: 14px;
+  color: #666;
+}
```

Comment:
The class name `.day` does not match the `calendar__day` class used in your HTML. Update the class name to ensure styles are applied correctly.